hail/src/main/scala/is/hail/expr/ir/functions/ArrayFunctions.scala

```diff
@@ -8,6 +8,19 @@ import is.hail.expr.types.virtual._
 import is.hail.utils._
 
 object ArrayFunctions extends RegistryFunctions {
+  val arrayOps: Array[(String, Type, Type, (IR, IR) => IR)] =
```

Comment:
move back? This didn't change, right?

Reply:
I moved this out of the `registerAll` method so I could register the same functions for NDArrays.

Reply:
ah, got it.